libs/core/device-select/components/drawer-device.component.tsx

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import * as React from "react"
+import { FunctionComponent } from "Core/core/types/function-component.interface"
+import { DeviceImage } from "Core/overview/components/device-preview/device-image.component"
+import Text, {
+  TextDisplayStyle,
+} from "Core/__deprecated__/renderer/components/core/text/text.component"
+import styled, { css } from "styled-components"
+import {
+  backgroundColor,
+  borderColor,
+  borderRadius,
+  textColor,
+} from "Core/core/styles/theming/theme-getters"
+import { DeviceType } from "Core/device"
+import { CaseColor } from "Core/device/constants"
+import { getDeviceTypeName } from "Core/discovery-device/utils/get-device-type-name"
+import { intl } from "Core/__deprecated__/renderer/utils/intl"
+
+const Device = styled("div")<{ active: boolean }>`
+  display: flex;
+  justify-content: flex-start;
+  margin: 1rem 0.5rem 1rem 0.5rem;
+
```

Comment:
🟡 

Your code was missing animations for the hover. Introduced the transition property for smoother animations. Now, changes in background color occur gradually and more smoothly when hovering.

```suggestion
  margin: 1rem 0.5rem 1rem 0.5rem;
  transition: background-color ${transitionTime("veryQuick")} ${transitionTimingFunction("smooth")};
```

Reply:
I am not missing that, there is no mentioning this in Figma

Reply:
https://github.com/mudita/mudita-center/pull/1666#pullrequestreview-1836090683